.github/workflows/e2e.yml

```
@@ -0,0 +1,33 @@
+name: 'E2E Tests'
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

あとここもキャッシュされてなくて低速になってる気がするので、こんな感じで書くといいかも
```suggestion
        uses: actions/setup-node@v2
        with:
          node-version: '16'
          cache: 'yarn'
          cache-dependency-path: frontend/octavio/yarn-lock.json
```